phc/easy/util/api_cache.py

```diff
@@ -36,9 +37,13 @@ def filename_for_fhir_dsl(query: dict):
             json.dumps(query).encode("utf-8")
         ).hexdigest()[0:8]
 
+        path_name = [
+            c for c in query.get("path", "").split("/") if "-" not in c
```

Comment:
We want part of the cached filename to include the path (but not the long, unintuitive UUIDs). See the example later on of the `genomics/projects/.../tests` call.

Reply:
https://github.com/lifeomic/phc-sdk-py/pull/112/files#diff-63ff39fb53d411813d938ec712d5716caee4352d5d5d1e02b5955fad01486550R6